request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestController.java

```diff
@@ -59,7 +59,7 @@
  */
 public class RequestController implements Service<RequestController>, ServerActivity {
 
-    public static final ServiceName SERVICE_NAME = ServiceName.JBOSS.append("server", "global-request-controller");
+    public static final ServiceName SERVICE_NAME = RequestControllerRootDefinition.REQUEST_CONTROLLER_CAPABILITY.getCapabilityServiceName();
```

Comment:
This is fine, although it would be better if this constant was deprecated and then moved to internal use only once the uses of it in full have shifted to using the capability instead.

In a case or two we've done as James suggested kept the old ServiceName around and registered the service under that name as an alias as a help for folks who may not have used the old constant. But I don't think that's needed here; the request controller stuff is new code without time for old crufty external user code to build up around it.